UIKitDZ/UIKitDZ/Controllers/BillViewController.swift

```diff
@@ -0,0 +1,137 @@
+//
+//  BillViewController.swift
+//  UIKitDZ
+//
+//  Created by 2lup on 28.09.2022.
+//
+
+import UIKit
+
+/// BillViewController: Экран оплаты заказа
```

Comment:
```suggestion
/// Экран оплаты заказа
```
BillViewController:  - это я вижу ниже , писать не нужно